tools/swift-plugin-server/Sources/CSwiftPluginServer/include/PluginServer.h

```diff
@@ -32,10 +92,10 @@ const void *PluginServer_createConnection(const char **errorMessage);
 void PluginServer_destroyConnection(const void *connHandle);
 
 /// Read bytes from the IPC communication handle.
-size_t PluginServer_read(const void *connHandle, void *data, size_t nbyte);
+SwiftUInt PluginServer_read(const void *connHandle, void *data, SwiftUInt nbyte);
 
 /// Write bytes to the IPC communication handle.
-size_t PluginServer_write(const void *connHandle, const void *data, size_t nbyte);
+SwiftUInt PluginServer_write(const void *connHandle, const void *data, SwiftUInt nbyte);
```

Comment:
`read` and `write` can return negative values, so I believe we should use `SwiftInt` for the return values. For arguments, I don't really care, but you changed them to cast to `Int`  https://github.com/apple/swift/pull/68268/files#diff-767b2b406807fd466b462821427b84772bdb1ddd27f3bea3781df7ec490ef2daL175-R196 So I _think_ this should be `SwiftInt` as well?